modules/insticatorBidAdapter.js

```diff
@@ -178,6 +190,48 @@ function buildImpression(bidRequest) {
     imp.video = buildVideo(bidRequest);
   }
 
+  if (isFn(bidRequest.getFloor)) {
+    let moduleBidFloor;
+
+    const mediaType = deepAccess(bidRequest, 'mediaTypes.banner') ? 'banner' : deepAccess(bidRequest, 'mediaTypes.video') ? 'video' : undefined;
+
+    let _mediaType = mediaType;
+    let _size = '*';
+
+    if (mediaType && ['banner', 'video'].includes(mediaType)) {
+      if (mediaType === 'banner') {
+        const { w: width, h: height } = imp[mediaType];
+        if (width && height) {
+          _size = [width, height];
+        }
+      } else {
+        const sizes = deepAccess(bidRequest, 'mediaTypes.banner.format');
+        if (sizes && sizes.length > 0) {
+          const [width, height] = sizes[0];
+          _size = [width, height];
+        }
+      }
+    } else if (mediaType === 'video') {
+      const { w: width, h: height } = imp[mediaType];
+      _mediaType = mediaType;
+      _size = [width, height];
+    }
+    try {
+      moduleBidFloor = bidRequest.getFloor({
+        mediaType: _mediaType,
+        size: _size
+      });
+    } catch (err) {
+      // continue with no module floors
+      logWarn('priceFloors module call getFloor failed, error : ', err);
+    }
+
+    if (moduleBidFloor && moduleBidFloor.currency === 'USD') {
```

Comment:
prebid offers a currency conversion module. With this code you effectively disable floors for everybody that doesn't use USD, even though it should work.

See https://docs.prebid.org/dev-docs/modules/floors.html#currency

Reply:
@muuki88 Updated the PR. Please check now.

I have added the default currency as `USD`.

Reply:
Nice 👍 LGTM